unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -5421,11 +5390,9 @@ private IEnumerator checkDropHandObjectActionFast(SimObjPhysics currentHandSimOb
                 }
             }
 
-            if (!success) {
-                errorMessage = "Object failed to open/close successfully.";
+            if (markActionFinished) {
+                actionFinished(errorMessage.Length == 0);
             }
```

Comment:
This might be slightly dangerous if some method that calls this had already set the `errorMessage` to something.

Reply:
I agree - i would prefer that we explicitly set whether actions are successful or not.  Relying on the errorMessage string may lead to erroneous failures.

Reply:
fixed